docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '1'
```

Comment:
Do you need version 1 here? It might be nice to go with version 2 if possible since this is a new project.